backend/src/openarchiefbeheer/templates/admin/base_site.html

```diff
@@ -8,7 +8,7 @@
 {% block extrastyle %}
     {{ block.super }}
 
-    <link rel="stylesheet" type="text/css" href="{% static 'css/admin_overrides.css' %}">
```

Comment:
Do we want to limit this directory name to only css?

Reply:
At the moment we only have the css overrides :thinking: So I think we can limit it to css?